javascriptv3/example_code/cross-services/textract-react/DeployDestroy.js

```diff
@@ -0,0 +1,140 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+/**
+ * This script deploys and destroys additional resources for the example after the
+ * primary resources have been deployed by running the setup.yaml AWS CloudFormation
+ * script.
+ *
+ * Before running this script, deploy the CloudFormation stack by running the following
+ * at a command prompt:
+ *
+ *     aws cloudformation create-stack --stack-name <stack-name> --template-body
+ *     file://setup.yaml --capabilities CAPABILITY_IAM
+ *
+ * ** Deploy **
+ * Deploy the additional demo resources by running the following at a command prompt:
+ *
+ *     node DeployDestroy.js deploy <stack-name>
+ *
+ * Remember to substitute the name of the CloudFormation stack for `<stack-name>`.
+ *
+ * ** Destroy **
+ * Destroy the additional demo resources by running the following at a command prompt:
+ *
+ *     node DeployDestroy.js destroy
+ *
+ * Remember to destroy the CloudFormation stack after running this script to completely
+ * remove resources created for the demo.
+ */
+
+import fs from 'fs';
+import {Config} from "./src/Config.js";
```

Comment:
Optional: run [prettier](https://prettier.io/)

Reply:
I tried it and I like it!